src/Sentry/buildTransitive/Sentry.targets

```diff
@@ -12,12 +12,17 @@
           Outputs="$(IntermediateOutputPath)$(SentryAttributesFile)">
     <PropertyGroup>
       <SentryAttributesFilePath>$(IntermediateOutputPath)$(SentryAttributesFile)</SentryAttributesFilePath>
+      <RepoRoot>$([System.IO.Path]::GetDirectoryName($([MSBuild]::GetPathOfFileAbove('.gitignore', '$(ProjectDir)'))))</RepoRoot>
```

Comment:
any folder can have `.gitignore` and in monorepos this can be more common.
How about calling this `GitIgnorePath` instead?